src/test/java/uk/nhs/hee/tis/revalidation/connection/controller/ConnectionControllerTest.java

```diff
@@ -317,15 +317,20 @@ void shouldReturnConnectedTraineeDoctorsInformation() throws Exception {
     final var connectionSummary = prepareConnectionSummary();
     final var pageableAndSortable = PageRequest.of(Integer.parseInt(PAGE_NUMBER_VALUE), 20,
         by(ASC, "gmcReferenceNumber.keyword"));
-    when(connectedElasticSearchService.searchForPage(EMPTY_STRING, pageableAndSortable))
+    final String dbc1 = "1-AIIDHJ";
+    final String dbc2 = "AIIDMQ";
+    final String dbcformatted = "aiidhj aiidmq";
```

Comment:
Ah yes I think this is because I started with the implementation of the formatting of dbcs so that the es query worked in this class, then moved it out later